src/App.js

```diff
@@ -0,0 +1,21 @@
+import "./App.css";
+
+const lista_uzytkownikow = [
+  { imie: "Kinga" },
+  { imie: "Mateusz" },
+  { imie: "Wiktoria" },
+  { imie: "Tomasz" },
+];
+console.log(lista_uzytkownikow);
```

Comment:
Kasia po co dajesz console.log z listy użytkowników!!!!!????? Przecież to wyciek danych!!!!!!!!